cpp/src/decisiontree/batched-levelalgo/builder_base.cuh

```diff
@@ -338,13 +338,16 @@ struct Builder {
     raft::update_device(curr_nodes, h_nodes.data() + node_start, batchSize, s);
 
     int total_samples_in_curr_batch = 0;
+    int n_large_nodes_in_curr_batch = 0;
```

Comment:
Comment what this variable is. e.g. nodes with number of training instances larger than block size. These nodes require global memory for histograms.

Reply:
done 👍🏾